backend/typescript/services/implementations/camperService.ts

```diff
@@ -234,6 +235,7 @@ class CamperService implements ICamperService {
           contactEmail: camper.contactEmail,
           contactNumber: camper.contactNumber,
           campSession: camper.campSession ? camper.campSession.toString() : "",
+          status: "NOT REGISTERED",
```

Comment:
should be waitlistedCamper.status here? they could have different statuses

```suggestion
          status: camper.status,
```